package.json

```diff
@@ -1,7 +1,7 @@
 {
   "type": "module",
   "dependencies": {
-    "@slflows/sdk": "^0.0.8",
+    "@slflows/sdk": "^0.2.0",
```

Comment:
We skipped a minor?

Reply:
Here, yes, but we did publish 0.1.0 and 0.1.1 in the meantime, earlier today. But I made my breaking changes another minor.